src/main/java9/module-info.java

```diff
@@ -43,7 +43,7 @@
   requires static net.bytebuddy;
```

Comment:
Do we want to also apply this to net.bytebuddy? Since I believe this is needed for the assumption mechanism to work correctly.

```suggestion
  requires net.bytebuddy;
```